src/components/atoms/headers/Header.tsx

```diff
@@ -1,3 +1,46 @@
-export function Header(){
-  return <div></div>
-}
\ No newline at end of file
+import styled from '@mui/system/styled';
+
+const HeaderContainer = styled('div')({
+  margin: '20px',
+  fontWeight: 'bold',
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  color: 'white'
+});
+
+const TitleContainer = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+});
+
+const FirstName = styled('p')(({ theme }) => ({
+  margin: 0,
+  fontSize: '50px',
+  [theme.breakpoints.up('lg')]: {
+    fontSize: '65px', 
+  },
+}));
+
+const LastName = styled('p')(({ theme }) => ({
+  margin: 0,
+  fontSize: '15px',
+  marginTop: '-10px',
+  [theme.breakpoints.up('lg')]: {
+    fontSize: '20px', 
+  },
+}));
+
+const Header = () => {
+  return (
+    <HeaderContainer>
+      <TitleContainer>
+        <FirstName>KURT</FirstName>
+        <LastName>SENSENBRENNER</LastName>
+      </TitleContainer>
+    </HeaderContainer>
+  );
+};
+
+export default Header;
```

Comment:
I thought that all React components are either written `export default function() {` or the `export default Header;` at the end of the file. I was taught the latter way at Flatiron. 

Which way is the 51% that you prefer not to have?

Reply:
Oh, I think I just got what you're saying. 
You prefer this syntax:
```
export function VideoList(){
  return <div></div>
}
```

Versus the `export default VideoList` syntax at the bottom of a file?

Reply:
Right. I prefer not having a `default` export.  51%.

Reply:
Ok, for learning purposes, why do you prefer not having it written that way?

Reply:
There's not really a technical reason for this.  Either work. Both are fine. The main think I like is consistency.

I prefer this way because it encourages us to be more "explicit". Rather than some default variable, we're exporting a named variable. 

Also, I think it looks cleaner if we have to import other types from the file:

```typescript
import Header, { HeaderType } from './header'
```

vs

```typescript
import { Header, HeaderType } from './header'
```

There's not really a benefit one way or the other. Just preference. 

But I do care about consistency. A lot.

Reply:
Great! Thanks for the explanation!
Makes perfect sense and I completely agree on the consistency. Fun to learn each of our syntax preferences/knowledge. 

I'll go through tomorrow and make sure all components are:

`export function Name() {
   return
}
`

And remove all:
`default export Name()`